core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/json.kt

```diff
@@ -107,8 +111,8 @@ public class JSON(
     }
 }
 
-public const val arrayColumnName: String = "array"
-public const val valueColumnName: String = "value"
+internal const val arrayColumnName: String = "array"
```

Comment:
They were made public for external modules like OpenAPI I believe. Though they could probably be renamed to ARRAY_COLUMN_NAME for consistency

Reply:
That's a good idea :) I think I originally made it public because user-side generated code could reference it, but I cannot find that anymore.